collections/README.md

```diff
@@ -112,6 +112,20 @@ const distinctNumbers = distinct(numbers);
 console.assert(distinctNumbers === [3, 2, 5]);
 ```
 
+### dropWhile
+
+Drop all elements in the given collection until the first element that does not
+match the given predicate.
```

Comment:
This should make clear that it does not touch the original array. We should also not use the word "collection" - it just takes an Array right now :-)